server/src/internalClusterTest/java/org/elasticsearch/reservedstate/service/FileSettingsServiceIT.java

```diff
@@ -502,7 +503,19 @@ public void testSettingsAppliedOnMasterReElection() throws Exception {
         assertClusterStateSaveOK(savedClusterState.v1(), savedClusterState.v2(), "43mb");
     }
 
-    public void testHealthIndicator() throws Exception {
+    public void testHealthIndicatorWithSingleNode() throws Exception {
+        internalCluster().setBootstrapMasterNodeIndex(0);
+        logger.info("--> start the node");
+        String nodeName = internalCluster().startNode((dataNode()));
```

Comment:
I think you'll have to replace this with
```suggestion
        logger.info("--> start the node");
        String nodeName = internalCluster().startNode();
```
otherwise there's no master node in the cluster.

Reply:
Oh lol this got lost in an edit. It used to say `masterNode(dataNode())`.

Can I just say `startNode`? That's neat.